packages/react-core/src/components/OverflowMenu/examples/OverflowMenu.md

```diff
@@ -84,13 +87,21 @@ class SimpleOverflowMenu extends React.Component {
         <OverflowMenuControl>
           <Dropdown
             onSelect={this.onSelect}
-            toggle={<KebabToggle onToggle={this.onToggle} />}
+            toggle={toggleRef => (
+              <MenuToggle
+                ref={toggleRef}
+                aria-label="kebab dropdown toggle"
```

Comment:
Just to avoid the multiple buttons with the same label inside of VoiceOver's rotor menu:

```suggestion
                aria-label="Simple example overflow menu"
```